lib/api.ts

```diff
@@ -248,29 +268,30 @@ function getCalendarData(activity: Activity[]) {
   });
 }
 
-const HIGHLIGHT_KEYS = [
-  "eod_update",
-  "comment_created",
-  "pr_opened",
-  "pr_reviewed",
-  "pr_merged",
-  "pr_collaborated",
-  "issue_assigned",
-  "issue_opened",
-] as const;
+// const HIGHLIGHT_KEYS = [
+//   "eod_update",
+//   "comment_created",
+//   "pr_opened",
+//   "pr_reviewed",
+//   "pr_merged",
+//   "pr_collaborated",
+//   "issue_assigned",
+//   "issue_opened",
+// ] as const;
 
-const computePoints = (
-  calendarDataEntry: Highlights,
-  initialPoints: number,
-) => {
-  return HIGHLIGHT_KEYS.map(
-    (key) => points[key] * (calendarDataEntry[key] ?? 0),
-  ).reduce((a, b) => a + b, initialPoints);
-};
+// const computePoints = (
+//   calendarDataEntry: Highlights,
+//   initialPoints: number,
+// ) => {
+//   return HIGHLIGHT_KEYS.map(
+//     (key) => points[key] * (calendarDataEntry[key] ?? 0),
+//   ).reduce((a, b) => a + b, initialPoints);
+// };
 
 const HighlightsReducer = (acc: Highlights, day: Highlights) => {
   return {
-    points: computePoints(day, acc.points),
+    // points: computePoints(day, acc.points),
+    points: acc.points + (day.points ?? 0),
```

Comment:
instead of changing this, tweak the `computePoints` instead.

Let's make computePoints more generic (being able to compute points dynamically based on conditions etc. instead of simply having a lookup to the `points` map).